js/shared/core.js

```diff
@@ -171,8 +188,21 @@ function unix_open(path, flags, perm) {
     is_wronly = true;
   }
 
+  let file;
   /* inlined from caml_sys_open */
-  var file = root.device.open(root.rest, f);
+
+  // Node is weird and doesn't like opening NUL on Windows
+  // if its fs.openSync("NUL", "rs+") call, so we have to
+  // special case it and use \\\\.\\nul instead.
+  if (globalThis.process.platform === "win32" && path == "NUL") {
+    path = "\\\\.\\nul";
```

Comment:
```suggestion
    path = {|\\.\nul|};
```
Think you can do this instead?

Reply:
Sorry. This is js, not ocaml. Could do `` String.raw`<string>` ``.

Reply:
ignoring this lmao